### ISSUE TITLE
Code should use threads to run in parallel

### ISSUE THREAD

**Issue by dirkgr** [allenai/bff#1]
You should be able to run in parallel by running multiple processes right now, but it might be faster/more accurate to run multiple threads in one process.

This necessitates some way of having each thread write to a different output file though, so it's not totally trivial.

**Comment by dirkgr** [comment]
This is already done.
